docs/guides/relocation-guide.md

```diff
@@ -80,8 +80,31 @@ duffle install forge.json --bundle-is-file --relocation-mapping relmap.json ...
 
 ```
 
-The invocation image then installs the software such that its images are loaded from the internal registry.
+The invocation image is loaded from the internal registry and installs the software such that its images are also loaded from the internal registry.
 
 ### Thick Bundle Relocation
 
-Not yet supported.
+Gringotts Wizarding Bank (GWB) needs to install some software into a new coin sorting machine.
+For GWB, security is paramount. Like Acme, all their production software must be loaded from internal repositories.
+However, GWB regard a networked DMZ as too insecure. Their data center has no connection to the external internet.
+
+Software is delivered to GWB encoded in Base64 and etched on large stones which are then rolled by hand into the
+GWB data center, scanned, and decoded. The stones are stored for future security audits.
+
+GWB obtains the new software as a thick bundle (`sort.tgz`) and relocates it to their private registry as follows:
+```bash
+duffle relocate sort.tgz --bundle-is-file --repository-prefix=registry.gold.gwb.dia/griphook --relocation-mapping relmap.json
```

Comment:
Ah, `--repository-prefix` determines the private registry that the bundle's images should be relocated to, but `--relocation-mapping` is the path of a file which is _output_ by the relocate operation.

Since this was confusing, I added a paragraph to explain it in the  "duffle relocate" section earlier in the guide.